utils/node_local_test.py

```diff
@@ -437,23 +453,15 @@ def stop(self):
         assert rc.returncode == 0
 
         start = time.time()
+        max_stop_time = 1
         while True:
             time.sleep(0.5)
-            rc = self.run_dmg(['system', 'query'])
-            ready = False
-            if rc.returncode == 0:
-                for line in rc.stdout.decode('utf-8').splitlines():
-                    if line.startswith('status'):
-                        if 'Stopped' in line:
-                            ready = True
-                        if 'Stopping' in line:
-                            ready = True
-
-            if ready:
-                break
-            if time.time() - start > 20:
-                print('Failed to stop')
+            if self._check_system_state('stopped'):
```

Comment:
Last time I checked this it never got past stopping, but perhaps that's that the two numerical values in _check_system_state() are for?

Reply:
Yeah this seems to be an intermittent issue that @tanabarr and I need to look into. I figured that being able to specify that stopping|stopped are the same state for the purposes of this test would be fine for now though.